dflag/dyngeneric.go

```diff
@@ -132,46 +134,34 @@ func (d *DynValue[T]) Get() T {
 	return d.av.Load().(T)
 }
 
-// ComaStringToSlice converts a coma separated string to a slice.
-func ComaStringToSlice(input string) []string {
+// CommaStringToSlice converts a coma separated string to a slice.
+func CommaStringToSlice(input string) []string {
 	// originally the heavy handed csv.NewReader(strings.NewReader(input)).Read()
 	return strings.Split(input, ",")
 }
 
 // Parse converts from string to our supported types (it's the missing generics strconv.Parse[T]).
 func Parse[T any](input string) (val T, err error) {
```

Comment:
If you unexport this, you can write an exported wrapper that has the correct type constraint and calls the internal version, so you get the type safety you'd like while still enabling `DynValue` to call it.

```
func Parse[T DynValueTypes](input string) (T, error) {
	return parse[T](input)
}
```

Reply:
yeah I just was just doing that but had to tweak the tests/add one - done now, thanks!